src/backend/utils/add-feed.js

```diff
@@ -0,0 +1,49 @@
+require('../lib/config');
+
+const args = require('minimist')(process.argv.slice(2));
+const { logger } = require('./logger');
+const { isValidUrl } = require('./utils');
+const feedQueue = require('../feed/queue');
+
+const log = logger.child({ module: 'add-feed' });
+
+async function add() {
+  if (
+    !Object.prototype.hasOwnProperty.call(args, 'name') ||
+    !Object.prototype.hasOwnProperty.call(args, 'url')
+  ) {
+    log.error('INVALID NUMBER OF ARGUMENTS');
```

Comment:
Nicer to print a help/usage message when the args are wrong, teach the user how to call it.

Reply:
good idea!